### ISSUE TITLE
Vue tutorial issues

### ISSUE THREAD

**Issue by vhristov5555** [NativeScript/playground-tutorials#19]
- In every block of code we need to specify the type of code that is displayed for examples:
![image](https://user-images.githubusercontent.com/7846198/44347711-36bf5980-a4a1-11e8-91c7-d9dd2ada1aef.png)
this would format the code: 
![image](https://user-images.githubusercontent.com/7846198/44347774-65d5cb00-a4a1-11e8-9100-47ff5f151947.png)
-  We need to format the code to be align correctly (see the above examples image property is shift to the right)
- Lesson 3: Step 4: we need to separate the actions in three different actions each for every code line. In step we should give the info about the code and where to copy it. In the moment in the first action we are telling them to copy the lines of code but the code is after the thirds step, which is confusing.
![image](https://user-images.githubusercontent.com/7846198/44348559-6b341500-a4a3-11e8-9314-83313744cf27.png)
- Lesson 3 - step 5 we need to add `;` to several lines of code:
![image](https://user-images.githubusercontent.com/7846198/44349010-b4389900-a4a4-11e8-8859-6897bdd6afa7.png)
Also in the same action we need to tell people that they need to add `,` to the last `}` before pasting the code.
- Lesson 4- step 7: We need to add steps for creating images folder and fronts folder. After that we should tell them upload the resources by clicking on the burger icon for each folder(images and fonts) and uploading the resources:
![image](https://user-images.githubusercontent.com/7846198/44349588-05955800-a4a6-11e8-84fb-8a84b7b9b2fa.png)

**Comment by vhristov5555** [comment]
We have found a bug in the Monaco Editor which leads to invalid code:
If you copy paster a code starting with a comment it is going to generate an invalid code. For example:
```
created(){
//this.getMyWeather()
var currentDate = new Date()
```
The Monaco editor will generate this code after copy paste:
```
created(){
//this.getMyWeather() var currentDate = new Date()
```
We have several steps including such kind of behaviour, in such way we need to modify them:
- Remove the comment code
- Add a step for adding the code instead of uncomment the code

**Comment by tjvantoll** [comment]
Hey @vhristov5555,

I’ve addressed all your comments except one in #23. There’s one additional bug with the Monaco editor that I’m really struggling to work around. I put a TODO in [here](https://github.com/NativeScript/playground-tutorials/blob/weather-fixes/cleanWeather-vue/3.md) to show you the spot, but essentially if I put a `then()` after the `http.request()` call in `getMyWeather()` the editor refuses to format the code.

Do you have any idea what’s going on here because it’s confusing the heck out of me? The good news is all the other issues you described should be resolved.

**Comment by tjvantoll** [comment]
I pushed another commit to workaround the problem. We should be good to go.

**Comment by tjvantoll** [comment]
Closing as we addressed these in #23.
